### ISSUE TITLE
MDN web docs - Setting up a Node development environment

### ISSUE THREAD

**Issue by Bjeurn2** [mdn/kuma#6408]
Dear sir/madame,
Thank you very much for your excellent tutorials about almost everything on the web. 
I am however confused about certain aspects of your tutorials, for instance your introductionary paragraphs about how to load Node.js on Windows.  Here is a quote that confuses me right from the start: 
_quote
macOS
Installing Node and NPM on Windows and macOS is straightforward because you can just use the provided installer:
    Download the required installer:
        Go to https://nodejs.org/en/
        Select the button to download the LTS build that is "Recommended for most users".
    Install Node by double-clicking on the downloaded file and following the installation prompts.
Windows
The recommended way to install Node and NPM on Windows is to use the Windows Subsystem for Linux v2 (WSL2). Currently this requires installing a Windows Insider fast-track  etc...._
Unquote 
So, what is it? installing Node on Windows: using the provided installer or go through WSL2? (which, by the way, does not work on my Acer Laptop - I cannot upgrade my Windows 10 to the required build). Since straight forward installing Node is a lot easier (and less frustrating!) then through WSL 2, what is the gain from going the hard way?
Yours sincerely, 
Raggy Campagne

**Comment by peterbe** [comment]
Which tutorial are you referring to? What is the URL?

**Comment by chrisdavidmills** [comment]
@Bjeurn2 is referring to https://wiki.developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/development_environment

I've updated it to remove the complex Windows bit — I don't know why that has been added, because it certainly makes things enormously more complex. I suspect it is because Node is more reliable on Linux, or the terminal commands are easier to explain. But it shouldn't make that much difference. Let me know if you do run into problems.
